Visualizer-Web-App/src/app/app.component.html

```diff
@@ -18,18 +18,20 @@
                 <div *ngIf="toggle" class="col s3 m3 l3"><youtube-dl></youtube-dl></div>
                 <div *ngIf="!toggle" class="col s3 m3 l3"><file-upload></file-upload></div>
             </td>
-            <td class="col s3 m3 l3 greyed-out-fingerprint fixed-action-btn click-to-toggle">
-                <a class="btn-floating btn-large waves-effect waves-blue waves-light blue" >
-                    <i class="material-icons">fingerprint</i>
-                </a>
-                <ul>
-                    <li><a class="btn-floating btn-large waves-effect waves-red red pulse" onclick="Materialize.toast('Really disliked...', 3000, 'rounded')"><i class="material-icons">sentiment_very_dissatisfied</i></a></li>
-                    <li><a class="btn-floating btn-large waves-effect waves-orange orange pulse" onclick="Materialize.toast('Disliked...', 3000, 'rounded')"><i class="material-icons">sentiment_dissatisfied</i></a></li>
-                    <li><a class="btn-floating btn-large waves-effect waves-yellow yellow darken-1 pulse" onclick="Materialize.toast('Liked...', 3000, 'rounded')"><i class="material-icons">sentiment_satisfied</i></a></li>
-                    <li><a class="btn-floating btn-large waves-effect waves-green green pulse" onclick="Materialize.toast('Loved...', 3000, 'rounded')"><i class="material-icons">sentiment_very_satisfied</i></a></li>
-                    <li><slide-menu></slide-menu></li>
-                </ul>
-            </td>
+
         </tr>
     </table>
 </div>
+
+<div class="greyed-out-fingerprint fixed-action-btn click-to-toggle">
+    <a class="btn-floating btn-large waves-effect waves-blue waves-light blue" >
+        <i class="material-icons">fingerprint</i>
+    </a>
+    <ul>
+        <li><a class="btn-floating btn-large waves-effect waves-red red pulse" onclick="Materialize.toast('Really disliked...', 3000, 'rounded')"><i class="material-icons">sentiment_very_dissatisfied</i></a></li>
+        <li><a class="btn-floating btn-large waves-effect waves-orange orange pulse" onclick="Materialize.toast('Disliked...', 3000, 'rounded')"><i class="material-icons">sentiment_dissatisfied</i></a></li>
+        <li><a class="btn-floating btn-large waves-effect waves-yellow yellow darken-1 pulse" onclick="Materialize.toast('Liked...', 3000, 'rounded')"><i class="material-icons">sentiment_satisfied</i></a></li>
+        <li><a class="btn-floating btn-large waves-effect waves-green green pulse" onclick="Materialize.toast('Loved...', 3000, 'rounded')"><i class="material-icons">sentiment_very_satisfied</i></a></li>
+        <li><slide-menu></slide-menu></li>
+    </ul>
+</div>
```

Comment:
@Avin12892 Not sure what the conflict was that you were having prior to this commit, but when I pull these changes the slide menu is acting funny. It's functioning fine, but it automatically slides out on the first time that I click the fingerprint button. Then I have to click the slide menu button twice to hide it. After that initial thing though it seems to work perfectly until I refresh. Are you experiencing this too?